### PR TITLE
UML-927 Added a triage page on actor side  and UML-944 Viewer homepage change

### PR DESCRIPTION
**Pull request by SeemaMenon**
## Purpose
https://opgtransform.atlassian.net/browse/UML-927
https://opgtransform.atlassian.net/browse/UML-944

Fixes UML-927 and UML-944

## Approach

- Added a twig template for the triage page
- Added handler
- Corrected back links
- start page is redundant now
- Adding tests
- change the home page to `/home' for actor and viewer

Note: we had to do this together due to common behat test code that needed changing, with associated break fixes

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [X] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/features/actor-account-creation.feature [hunk 1]
```diff
@@ -27,13 +28,15 @@ Feature: Account creation
   Scenario: The user account creates an account which already exists
     Given I am not a user of the lpa application
     And I want to create a new account
+    Given I access the account creation page
```
Comment (cooperaj) [inline service-front/app/features/actor-account-creation.feature:31]: should be an *And*

File: service-front/app/features/actor-triage.feature [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  @ui
+  Scenario: The user can create new account page when they say they do not have an existing account
+    Given I am on the triage page
+    When I select the option to create a new account
+    Then I am allowed to create an account
+
+  @ui
+  Scenario: The user can navigate back to triage page from login page
+     Given I access the login form
+     When I click Back link on the page
```
Comment (cooperaj) [inline service-front/app/features/actor-triage.feature:24]: When I click the Back link on the page

**Comment by williamfalconeruk** [comment]
triage handler needs the `@codeCoverageIgnore`  annotation in place if we have Behat tests in place covering the page.

**Comment by williamfalconeruk** [comment]
Triage Form needs test in place.

File: service-front/app/features/context/UI/AccountContext.php [hunk 3]
```diff
@@ -3422,4 +3376,72 @@ public function theCorrectLPAIsFoundAndICanSeeTheCorrectNameWhichWillHaveARoleOf
...
+
+    /**
+     * @When /^I select the option to sign in to my existing account$/
+     */
+    public function iSelectTheOptionToSignInToMyExistingAccount()
+    {
+
+        $this->ui->assertPageAddress('/home');
+        $this->ui->assertPageContainsText('Use a lasting power of attorney');
+        $this->ui->selectOption('triageEntry', 'yes');
```
Comment (GemTay) [inline service-front/app/features/context/UI/AccountContext.php:3388]: you can remove the select option step assuming fill field works the same

File: service-front/app/features/context/UI/CommonContext.php [hunk 4]
```diff
@@ -259,13 +258,25 @@ public function iWantToUseMyLastingPowerOfAttorney()
...
     }
 
+    /**
+     * @Given /^I am able to logine$/
+     */
+    public function iAmAbleToLogin()
+    {
+        $this->ui->assertPageAddress('/home');
+        $this->ui->selectOption('triageEntry', 'yes');
+        $this->ui->fillField('triageEntry', 'yes');
```
Comment (GemTay) [inline service-front/app/features/context/UI/CommonContext.php:268]: selecting option twice here

**Pull request merged**